powersimdata/data_access/context.py

```diff
@@ -8,7 +12,7 @@ class Context:
     @staticmethod
     def get_data_access(data_loc=None):
         """Return a data access instance appropriate for the current
-        environment. Currently only supports internal client server mode.
+        environment.
 
         :param str data_loc: pass "disk" if using for backups otherwise leave
```

Comment:
should it read `pass "disk" if using backups otherwise ...`

Reply:
Should it? I wasn't sure on the wording, can change if that's more accurate

Reply:
Not sure, you are the native speaker! What about: pass "disk" if data are located on a mounted disk whose location is in specified in :mod:\`powersimdata.utility.server_setup\` ...

Reply:
Haha but you are more familiar with the backup feature :) The latest wording sounds good though